integration-test/uport-config.js

```diff
@@ -1,2 +1,2 @@
-import { Uport, PublicPersona, Registry } from 'uport-connect'
-export { Uport, PublicPersona, Registry }
```

Comment:
persona in uport-js? there is still persona functionality we will want once we have SD, profiles, sets of credentials mixed together,